client/core/core.go

```diff
@@ -3582,15 +3582,48 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 					notifyErr(SubjectMatchStatusError, "Match %s for order %s is in state %s, but has no maker swap contract.", dbMatch.Side, tracker.token(), dbMatch.Status)
 					continue
 				}
+				// Obtaining AuditInfo will fail if it's unmined AND gone from
+				// mempool, or the wallet is otherwise not ready. Note that this
+				// does not actually audit the contract's value, recipient,
+				// expiration, or secret hash (if maker), as that was already
+				// done when it was initially stored as CounterScript.
 				auditInfo, err := wallets.toWallet.AuditContract(counterSwap, counterContract)
 				if err != nil {
-					c.log.Debugf("AuditContract error for match %v status %v, refunded = %v, revoked = %v: %v",
-						match.id, match.MetaData.Status, len(match.MetaData.Proof.RefundCoin) > 0, match.MetaData.Proof.IsRevoked(), err)
-					match.swapErr = fmt.Errorf("audit error, order %s, match %s: %w", tracker.ID(), match.id, err)
-					match.MetaData.Proof.SelfRevoked = true
```

Comment:
This was the source of the secondary bug addressed by this PR.  Namely, by revoking it in memory, the order would be declared inactive and retired, but on restart this match would reload as active.  However, since the order was already retired and any change coins were returned, it was likely that the order would fail to load further down in `resumeTrades` when trying to find funding coins.  The result is seeing the funding coin error forever each time dexc is started.